test/test_graphs.py

```diff
@@ -61,3 +62,18 @@ def test_hello():
     edge = elements[2]
     check_edge(edge, hello, world)
     check_edge_dir(edge, dx=0, dy=1)
+
+
+def test_polylines():
+    file = "./undirected/polylines.gv.txt"
+    xml = graphviz2drawio.convert(file)
+    print(xml)
+
+    root = ET.fromstring(xml)
+    elements = check_xml_top(root)
```

Comment:
Looks like a much better way to check 👍 
But why remove the testing on the other `undirected`?

Reply:
what do you mean with remove other testing?
I would love to execute every test (at least to check that every .dot generates its own xml) but many of them do not work. Some, even require external images for example